README.md

```diff
@@ -266,6 +330,14 @@ public function getDropsEntitlements(array $parameters = [])
 public function redeemEntitlementsCode(array $parameters = [])
 ```
 
+### EventSub
+
+```php
+public function subscribeEventSub(array $parameters = [], array $body = [])
```

Comment:
I don't know yet if we want to leave the payload in there, officially you don't need it.